docs/installation/windows.md

```diff
@@ -135,13 +136,23 @@ it.
 
 DEV requires Redis version 4.0 or higher.
 
-We recommend to follow [this
-guide](https://redislabs.com/blog/redis-on-windows-10/) to run Redis under WSL.
+We recommend to follow
+[this guide](https://redislabs.com/blog/redis-on-windows-10/) to run Redis under
+WSL.
+
+### Elasticsearch
+
+DEV requires Elasticsearch version 7 or higher.
+
+We recommend following the install guide
```

Comment:
same here, let's recommend people to install the OSS version:

> Alternatively, you can download the following package, which contains only features that are available under the Apache 2.0 license: https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.5.2-windows-x86_64.zip

from https://www.elastic.co/guide/en/elasticsearch/reference/7.5/zip-windows.html#install-windows